sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -162,39 +163,39 @@ var data = new
 
 Response response = client.AnalyzeConversation(RequestContent.Create(data));
 
-using JsonDocument result = JsonDocument.Parse(response.ContentStream);
-JsonElement conversationalTaskResult = result.RootElement;
-JsonElement conversationPrediction = conversationalTaskResult.GetProperty("result").GetProperty("prediction");
+dynamic conversationalTaskResult = response.Content.ToDynamic();
```

Comment:
I defer to you on what the best thing for CLU customers is, but if you used `response.Content.ToDynamic(DynamicJsonOptions.AzureDefault)` here, you could use PascalCase property accessors in subsequent calls to make the sample code look more like it is using strongly-typed models.  An advantage of this is that if models are provided later, there is less refactoring needed to use them.

Reply:
Given input is camelCase, I still prefer output to also match the original (mostly) camelCase for consistency.